src/js/jsx/search/SearchBar.jsx

```diff
@@ -388,6 +390,10 @@ define(function (require, exports, module) {
                 return optionList.take(this.props.maxOptions);
             }
 
+            if (optionList.size === 6 && searchTerm !== "") {
```

Comment:
:police_car: :cop: :vertical_traffic_light:  We aren't allowed to log arbitrary strings, which I think is what's happening here with `searchTerm`. I also cannot abide by a magic number, 6, here without additional explanation. I think instead of this, you should just try to find the handler that is called when the search bar is called without executing any options, and log a generic `failed-search` event.